### PR TITLE
Feat remove duplicate title

### PR DESCRIPTION
**Pull request by SaketThota**
1. Made the main title hyperlink
2. Removed duplicate title

### CODE REVIEW

File: summary/combined-summary-template.html [hunk 1]
```diff
@@ -0,0 +1,468 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (SirusCodes) [inline summary/combined-summary-template.html:2]: WHAT? why this file?

File: summary/combined-summary.adoc [hunk 2]
```diff
@@ -1,6 +1,4 @@
 = OTC CatchUp Summaries
-Our Tech Community
```
Comment (SirusCodes) [inline summary/combined-summary.adoc:2]: It won't matter much but why are we removing this?

File: summary/individual-summary.adoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-= OTC CatchUp #{catchup_display_number} Summary
+= link:/summary/{catchup_display_number}[OTC CatchUp #{catchup_display_number} Summary]
```
Comment (SirusCodes) [inline summary/individual-summary.adoc:1]: Who would click on this link?

File: summary/combined-summary-template.adoc [hunk 4]
```diff
@@ -1,11 +1,12 @@
-[#{catchup_display_number}]
-=== link:/summary/{catchup_display_number}[OTC CatchUp #{catchup_display_number}]
+:leveloffset: -1
```
Comment (SirusCodes) [inline summary/combined-summary-template.adoc:1]: What this does?

File: summary/combined-summary-template.adoc [hunk 5]
```diff
@@ -1,11 +1,12 @@
-[#{catchup_display_number}]
-=== link:/summary/{catchup_display_number}[OTC CatchUp #{catchup_display_number}]
```
Comment (SirusCodes) [inline summary/combined-summary-template.adoc:2]: As far as I remember we don't need to make any changes in this

File: util/build.sh [hunk 6]
```diff
@@ -71,6 +71,7 @@ done;
 # Build combined summary site
 asciidoctor \
 	-a webfonts! \
+	-a building_combined_summary \
```
Comment (SirusCodes) [inline util/build.sh:74]: Also, what is this?🤔

**Review by SirusCodes — CHANGES_REQUESTED** [review]

File: summary/combined-summary-template.html (hunk 1 above)
Comment (KartikSoneji) [reply summary/combined-summary-template.html:2]: @SaketThota I don't think you meant to commit this, please amend your commits to remove it.

**Comment by SirusCodes** [comment]
@SaketThota are you still working on this?

**Comment by SaketThota** [comment]
> @SaketThota are you still working on this?

I think I fixed the problem of having double lines before _About OTC_ .
But now that you stated the changes, sure I can work on them and will raise a PR!

File: summary/combined-summary.adoc (hunk 2 above)
Comment (HarshKapadia2) [reply summary/combined-summary.adoc:2]: Yeah I don't think we should remove that line. That is the author of the page.

File: summary/individual-summary.adoc (hunk 3 above)
Comment (HarshKapadia2) [reply summary/individual-summary.adoc:1]: Yeah this is not required in the individual summary.

File: summary/combined-summary-template.adoc (hunk 5 above)
Comment (HarshKapadia2) [reply summary/combined-summary-template.adoc:2]: Yes, this needs to be kept in.

**Comment by HarshKapadia2** [comment]
Okay I am really confused. Is this PR still valid @SirusCodes?

**Comment by SirusCodes** [comment]
The issue is not fixed... So yeah it is valid.

@SaketThota you are still working on it?

**Comment by SaketThota** [comment]
I am not currently not working on it, but would like to take it up later.

**Comment by HarshKapadia2** [comment]
I think it would be better to open a new PR at this point. There are too many unnecessary changes here.

**Comment by tusharnankani** [comment]
What's the status/updates of this PR?

**Comment by HarshKapadia2** [comment]
@SaketThota @SirusCodes pinging y'all to look into this PR.

**Comment by SirusCodes** [comment]
I will handle it in #113

**Comment by HarshKapadia2** [comment]
Okay, then I'm closing this PR.

**Pull request closed without merging**